.github/workflows/ci.yml

```diff
@@ -65,4 +65,41 @@ jobs:
         with:
           subject-name: ${{ env.IMAGE_BASE }}
           subject-digest: ${{ steps.build.outputs.digest }}
-          push-to-registry: true
\ No newline at end of file
+          push-to-registry: true
+  test:
+    runs-on: ubuntu-latest
+    needs: build
+    timeout-minutes: 45
+    env:
+      CONTAINER_NAME: testnet4
```

Comment:
shouldn't we parametrize `CONTAINER_NAME` in order to add ability to test others?

Reply:
Sure it's designed to be turned into a matrix if/when we need it.